conf/services/tax-credits-renewals.yml

```diff
@@ -0,0 +1,11 @@
+serviceName: Tax Credits Renewals
+serviceHeaderName: Tax Credits Renewals service
+serviceDescription: |
+  The Tax Credits Renewals service can be used by anyone currently receiving tax credits to renew their claim during the
+  April to August renewals window.
+serviceDomain: www.tax.service.gov.uk
+serviceUrl: /tax-credits-service/renewals
+contactFrontendServiceId: tax-credits-renewals
+complianceStatus: noncompliant
+statementCreatedDate: 2021-03-22
+statementLastUpdatedDate: 2021-03-22
```

Comment:
1. This YAML is missing the mandatory property: `statementVisibility`
2. The property `serviceHeaderName` is no longer used and should be removed